src/ChainSafe.Gaming/RPC/Contracts/Contract.cs

```diff
@@ -161,20 +161,8 @@ public async Task<object[]> Send(string method, object[] parameters = null, Tran
             parameters ??= Array.Empty<object>();
 
             var function = contractBuilder.GetFunctionBuilder(method);
-            var txReq = overwrite ?? new TransactionRequest();
```

Comment:
I think this change breaks the original logic. The optional assignment for `From`, `To`, `Data` and etc was there in the case provided `overwrite` object does not have any.

So it was possible to provide some user-defined fields in this method. But with this logic `overwrite` object is not used at all.

Reply:
You're right! Fixed